src/js/core/methods/CardanoSignTransaction.js

```diff
@@ -27,6 +28,7 @@ const CardanoSignTransactionFeatures = Object.freeze({
     SignStakePoolRegistrationAsOwner: ['0', '2.3.5'],
     ValidityIntervalStart: ['0', '2.3.5'],
     MultiassetOutputs: ['0', '2.3.5'],
+    AuxiliaryDataAndCatalystRegistration: ['0', '2.3.7'],
```

Comment:
I'd name this just AuxiliaryData, as that's what the related logic is checking

Reply:
https://github.com/vacuumlabs/connect/pull/12/commits/62ab5cd64855bbb00fca699641fc8d3c8069d853#diff-3f63cf5f20614bf66b007cac352a997338d942f06ddec324b3eae61b8e1f3548R31